src/components/ActionBar/ActionBar.less

```diff
@@ -22,7 +22,7 @@
       color: var(--pacific-dark);
       font-size: @size-v;
       &:hover {
-        color: @link-text-hover;
+        color: var(--pacific-dark);
```

Comment:
I think we can get rid of all of these color definitions on L20, 22 and 25 since they should be standard button /link colors